.circleci/config.yml

```diff
@@ -26,14 +26,16 @@ executors:
         xpack.license.self_generated.type: trial
         xpack.security.authc.api_key.enabled: true
         ELASTIC_PASSWORD: password
-    - image: docker.elastic.co/enterprise-search/enterprise-search:7.8.0
+        ES_JAVA_OPTS: -Xms512m -Xmx512m
```

Comment:
This helps prevent the ES container from grinding to a halt during integration tests.